server.js

```
@@ -438,6 +476,128 @@ app.get('/api/admin/produits', checkAuth, checkAdmin, (req, res) => {
+        const inventaireContent = fs.readFileSync(inventairePath, 'utf8');
+        
+        // Extraire l'objet produitsInventaire du fichier JavaScript
+        const inventaireMatch = inventaireContent.match(/const produitsInventaire = ({[\s\S]*?});/);
+        if (!inventaireMatch) {
+            return res.status(500).json({ success: false, message: 'Format de fichier produitsInventaire invalide' });
+        }
+        
+        // Évaluer l'objet JavaScript de manière sécurisée
+        const inventaireObj = eval('(' + inventaireMatch[1] + ')');
+        
+        res.json({ success: true, produitsInventaire: inventaireObj });
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)